v2/internal/frontend/desktop/windows/win32/theme.go

```diff
@@ -50,6 +59,16 @@ func SetTheme(hwnd uintptr, useDarkMode bool) {
 	}
 }
 
+func EnableTranslucency(hwnd uintptr, backdrop BackdropType) {
+	fmt.Printf("%+v\n", windowsVersion)
```

Comment:
Should we remove this for the final merge?

Reply:
gone :-)